modules/Sources/Features/Root/RootStore.swift

```diff
@@ -297,11 +297,33 @@ extension AlertState where Action == RootReducer.Action {
         }
     }
     
-    public static func tmpMigrationToBeDeveloped() -> AlertState {
+    public static func differentSeed() -> AlertState {
         AlertState {
-            TextState("Automatic migration to be developed soon")
+            TextState("Warning")
+        } actions: {
+            ButtonState(role: .cancel, action: .alert(.dismiss)) {
+                TextState("Try Again")
+            }
+            ButtonState(role: .destructive, action: .initialization(.nukeWallet)) {
+                TextState("Continue")
+            }
+        } message: {
+            TextState("This recovery phrase doesn't match the Zashi database backup saved on this device. If you proceed, you will lose access to this database backup and if you try to restore later, some information may be lost.")
```

Comment:
In a follow-up, we could move these texts into the translation file so we can proofread and localize them once needed.